mozregression/fetch_build_info.py

```diff
@@ -154,6 +154,14 @@ def find_build_info(self, push):
             raise BuildInfoNotFound(
                 "unable to find a build url for the" " changeset %r" % changeset
             )
+
+        if self.fetch_config.app_name == "gve":
+            # Check taskcluster URL to make sure artifact is still around.
+            # build_url is an alias that redirects via a 303 status code.
+            status_code = requests.head(build_url, allow_redirects=True).status_code
+            if status_code != 200:
+                error = f"Taskcluster file {build_url} could not be fetched ({status_code})."
```

Comment:
Nit: we didn't really try to fetch the artifact, perhaps the error should specify we just checked if it existed. Would also be nice to indicate the value in brackets is a status code, ie `... fetched (status code: {status_code})`.